Add dataset 0002

simleo opened the pull request:
Adds the other dataset from [Masuzzo et al 2017](https://www.nature.com/articles/srep42383).

Also, using this to piggyback the removal of the isa-specific readme from dataset 0001 (now redundant after #6).

sbesson commented:
Closing/reopening to include #8

agbeltran commented:
As discussed with @simleo, the dataset included in this PR is the same as available at https://github.com/ISA-tools/ISAdatasets/tree/tests/tab/MIACME-Cellmissy2

Merging.

simleo commented:
Note that comparing this version of the dataset with https://github.com/ISA-tools/ISAdatasets/tree/tests/tab/MIACME-Cellmissy2 is not entirely straightforward due to the fact that I've adjusted whitespace. I have added a simple tool compare tabular files while ignoring part of the whitespace in #9.
